docs/README.md

```diff
@@ -0,0 +1,27 @@
+[[Back to main README](../README.md)]
```

Comment:
IMHO we should use Github to navigate through the project.

Reply:
see my comment above